lib/fs.js

```diff
@@ -276,7 +281,7 @@ function exists(path, callback) {
   }
 }
 
-ObjectDefineProperty(exists, internalUtil.promisify.custom, {
+ObjectDefineProperty(exists, promisify.custom, {
```

Comment:
If we're changing this line, we should use also fix that.
```suggestion
ObjectDefineProperty(exists, kInspect, {
```